NoahHusain/Truncheons-Flagons
# prevent users from submitting empty data fields in the new player/new team forms

New code checks if any fields are empty in the new player and new team forms and turns those fields red and stops the data from being sent to the API.  Additionally, if a team is not selected in the new player form, a message box will appear and prompt them to select a team.

steps to test
1.pull this branch and serve to the web browser
2. click the draft to team button without entering/selecting any information.  all empty fields should turn red and a message box should appear prompting the player to select a team.
3. enter data into some of the fields but not all.  choose a team.
4. fields in which data was entered should turn back to white, and fields that are empty should remain red.  the message box should not appear.  the leaderboard should appear unchanged (since no data should have been sent to the api)
5. enter data into all fields and select a team
6. the new player form should reset and the leaderboard should display the new player added to the selected team
7. click the create team without entering anything into the team name field
8. field should turn red and team should not be created
9. enter a team name and click on create team
10. new team form should reset and the leaderboard should display the new team added
